api/src/main/java/tennisclub/dto/UserAuthDTO.java

```diff
@@ -0,0 +1,25 @@
+package tennisclub.dto;
+
+public class UserAuthDTO {
+
+    private  String username;
```

Comment:
I think that when the user is authenticating, he/she should always provide a username and password - they shouldn't be blank. Therefore I would personally add a `@NotBlank` annotation 😄

Reply:
Oh, completely forgot the validation part, thanks!